net.sourceforge.scidvspc.yml

```diff
@@ -0,0 +1,52 @@
+app-id: net.sourceforge.scidvspc
```

Comment:
the website being `scidvspc.sourceforge.net` the app-id should be `net.sourceforge.scidvspc.scidvspc`